src/reducers/index.ts

```diff
@@ -1,11 +1,14 @@
-import { Reducer, combineReducers } from 'redux';
+import { combineReducers, Reducer } from "redux";
```

Comment:
Remove this unused import.

```suggestion
import { combineReducers } from "redux";
```